<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>346</width>
     <height>462</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbConnect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDisconnect">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="leHost">
        <property name="text">
         <string>naver.com</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lePort">
        <property name="text">
         <string>80</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="pteMessage"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPlainTextEdit" name="pteSend">
        <property name="plainText">
         <string>GET / HTTP/1.1
Host: naver.com

</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Send">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="cbUseSSL">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>92</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>UseSSL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbLoad">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>330</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Lode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSave">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>370</y>
     <width>87</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
